src/main/scala/com/databricks/labs/overwatch/utils/Tools.scala

```diff
@@ -617,22 +619,113 @@ object Helpers extends SparkSessionWrapper {
    *                        or direct access via s3:// or abfss:// or dbfs:/mnt/ etc.
    * @return
    */
+//  def registerRemoteOverwatchIntoLocalMetastore(
+//                                                 remoteWorkspace: Workspace,
+//                                                 localDataTarget: DataTarget
+//                                               ): Seq[WorkspaceMetastoreRegistrationReport] = {
+//
+//    val newConfigParams = remoteWorkspace.getConfig.inputConfig.copy(dataTarget = Some(localDataTarget))
+//    val newConfigArgs = JsonUtils.objToJson(newConfigParams).compactString
+//    val localTempWorkspace = Initializer(newConfigArgs, disableValidations = true)
+//    val registrationReport = localTempWorkspace.addToMetastore()
+//    val b = Bronze(localTempWorkspace, suppressReport = true, suppressStaticDatasets = true)
+//    val g = Gold(localTempWorkspace, suppressReport = true, suppressStaticDatasets = true)
+//
+//    b.refreshViews()
+//    g.refreshViews()
+//    registrationReport
+//
+//  }
   def registerRemoteOverwatchIntoLocalMetastore(
-                                                 remoteWorkspace: Workspace,
-                                                 localDataTarget: DataTarget
-                                               ): Seq[WorkspaceMetastoreRegistrationReport] = {
+                                                 remoteStoragePrefix: String,
+                                                 remoteWorkspaceID: String,
+                                                 localETLDatabaseName: String = "",
+                                                 localConsumerDatabaseName: String = "",
+                                                 remoteETLDataPathPrefixOverride: String = "",
+                                                 usingExternalMetastore: Boolean = false,
+                                                 workspacesAllowed: Array[String] = Array()
+                                               ): Seq[WorkspaceMetastoreRegistrationReport]  = {
+
+    // Derive eltDatapathPrefix
+    val eltDataPathPrefix = if (remoteETLDataPathPrefixOverride == "") {
+      remoteStoragePrefix + "/global_share"
```

Comment:
let's assert that remoteStoragePrefix does not end with `global_share` and alert the user that the storage prefix != the etlDataPathPrefix it's from Box 1 in the runner.